src/SmartComponents/AddSystemModal/AddSystemModal.js

```diff
@@ -92,7 +117,13 @@ class AddSystemModal extends Component {
                             eventKey={ 1 }
                             title="Baselines"
                         >
-                            <BaselinesTable hasSelect={ true }/>
+                            <BaselinesTable
+                                hasMultiSelect={ true }
```

Comment:
Is the default for BaselinesTable multi-select, single-select or no select?

Reply:
The default it whatever is passed to it. I actually needed to have removed the `hasMultiSelect` here. It makes it's decision based on the `tableType` passed to it.